openpype/hosts/resolve/utility_scripts/openpype_startup.scriptlib

```diff
@@ -0,0 +1,22 @@
+-- Run OpenPype's Python launch script for resolve
+function file_exists(name)
+   local f = io.open(name, "r")
+   return f ~= nil and io.close(f)
+end
+
+
+openpype_root = os.getenv("OPENPYPE_ROOT")
+if openpype_root ~= nil then
+    script = openpype_root .. "/openpype/hosts/resolve/startup.py"
```

Comment:
I would replce `OPENPYPE_ROOT` with resolve specific root, so the script is not based on location inside openpype.
```suggestion
resolve_root = os.getenv("OPENPYPE_RESOLVE_ROOT")
if resolve_root ~= nil then
    script = resolve_root .. "/startup.py"
```

This would require to set `OPENPYPE_RESOLVE_ROOT` e.g. in resolve addon.

```python
from .utils import RESOLVE_ROOT_DIR
...

class ResolveAddon(OpenPypeModule, IHostAddon):
    ...
    
    def add_implementation_envs(self, env, _app):
        env["OPENPYPE_RESOLVE_ROOT"] = RESOLVE_ROOT_DIR
```